openverse_catalog/dags/data_refresh/data_refresh_task_factory.py

```diff
@@ -137,38 +154,90 @@ def create_data_refresh_task_group(
             mode="reschedule",
             pool=DATA_REFRESH_POOL,
         )
+        tasks.append(wait_for_data_refresh)
 
-        data_refresh_post_data = {
-            "model": data_refresh.media_type,
-            "action": "INGEST_UPSTREAM",
-        }
+        # Get the index currently mapped to our target alias, to delete later.
+        get_current_stat = SimpleHttpOperator(
+            task_id="get_current_stat",
```

Comment:
For me `stat` is a bit confusing, could this maybe be `get_current_alias`?

Reply:
That's very valid. Since the task finds the name of the index that the alias refers to, I renamed it to `get_current_index` in a1408ff.